src/platforms/hubspot/actions/calls/list.ts

```diff
@@ -0,0 +1,29 @@
+import { transformCall } from '@/platforms/hubspot/actions/mappers';
+import { action } from '@/sdk';
+import { z } from 'zod';
+import client from '../../client';
+import { hubspotCommonAssociationsSchema } from '../../schemas';
+
+export default action(
+  'list-calls',
+  {
+    operation: 'list',
+    resource: 'calls',
+    mutation: false,
+    schema: z.object({
+      after: z.string().optional(),
+      pageSize: z.number().optional(),
+      associations: z.array(hubspotCommonAssociationsSchema).optional(),
+    }),
+    scopes: [],
+  },
+  async ({ auth, input }) => {
+    const result = await client.calls.list(auth, input);
+
+    return {
+      paging: result.data.paging,
+      results: result.data.results?.map(transformCall),
+      $native: result.$native,
```

Comment:
how come sometimes we spread the result from the client and other times we have these keys? does it make sense to make it more uniform?

Reply:
I spread it for the `finds` but not for the `lists`, which basically is how hubspot returns it to begin with. And it's also easier to apply the mapping without spreading it